aten/src/ATen/native/cpu/RangeFactoriesKernel.cpp

```diff
@@ -12,6 +13,30 @@ namespace {
 
 using namespace vec256;
 
+static void arange_kernel(TensorIterator& iter, Scalar scalar_start, Scalar scalar_steps, Scalar scalar_step) {
+  AT_DISPATCH_ALL_TYPES(iter.dtype(), "arange_cpu", [&]() {
+    using accscalar_t = at::acc_type<scalar_t, false>;
+    auto start = scalar_start.to<accscalar_t>();
+    auto steps = scalar_steps.to<accscalar_t>();
+    auto step = scalar_step.to<accscalar_t>();
+    at::parallel_for(0, steps, internal::GRAIN_SIZE, [&](int64_t p_begin, int64_t p_end) {
+      int64_t idx(p_begin);
+      TensorIterator it(iter);
+      cpu_serial_kernel_vec(
+          it,
+          [start, step, steps, &idx]() -> scalar_t {
+            return start + step * (idx++);
+          },
+          [start, step, steps, &idx]() -> Vec256<scalar_t> {
+            Vec256<scalar_t> res;
```

Comment:
Hello! Sorry it fails internal hard linters with

```
./aten/src/ATen/native/cpu/RangeFactoriesKernel.cpp:27:25: error: lambda capture 'steps' is not used [-Werror,-Wunused-lambda-capture]
```

Reply:
Thanks for pointing out this! Now fixed.